addon/components/s-button.js

```diff
@@ -4,5 +4,23 @@ import layout from '../templates/components/s-button';
 
 export default Component.extend(SpreadMixin, {
     classNames: ['s-button'],
-    layout
+    layout,
+    click() {
+      this.sendAction();
+      if (this.get('onClick')) {
+        this.get('onClick')();
+      }
+    },
+    actions: {
+      onMouseEnter() {
+        if (this.get('onMouseEnter')) {
+          this.get('onMouseEnter')();
```

Comment:
if arguments is passed?

Reply:
They are getting passed to the respective action.